content/02-getting-started/01-installing-the-cardano-node.mdx

```diff
@@ -17,6 +17,88 @@ the source code using either of the following:
 Once you have installed the node, you need to
 [specify the configuration parameters](https://github.com/input-output-hk/cardano-node/blob/master/doc/getting-started/understanding-config-files.md/).
 
+### How to Run the Cardano node and CLI using Docker
+There is also a Docker image available that you can use. 
+
+#### Setting up Docker
+
+1. Download and install Docker/Docker Desktop from the [Docker site](https://docs.docker.com/get-docker/).
+2. Ensure you have allocated enough RAM to Docker (we recommend setting the RAM to 8GB in the Docker Desktop requirements) and start the Docker daemon, as described in the Docker instructions.
```

Comment:
People may know what you mean but this step is only worth following when using Docker Desktop (available on Windows or Mac) because the default RAM limit is 2GB. Docker (on Linux) has no RAM limit for containers by default.